src/elm/arithmetic.coffee

```diff
@@ -206,7 +206,12 @@ module.exports.MinValue = class MinValue extends Expression
 
   exec: (ctx) ->
     if MIN_VALUES[@valueType]
-      MIN_VALUES[@valueType]
+      if @valueType == '{urn:hl7-org:elm-types:r1}DateTime'
+        minDateTime = MIN_VALUES[@valueType]
+        minDateTime.timezoneOffset = parseInt(ctx.getTimezoneOffset())
```

Comment:
Are we sure this will be an integer? I think CQL timezoneoffset is in hours, not minutes.

Reply:
Good point.  Since it won't affect Bonnie and we are under a tight time constraint, we can make that change for the `master` PR rather than do it here.

Reply:
Resolving for now, will address in master.